06_puppy/patrickmarabeas/store_test.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type StoreSuite struct {
+	suite.Suite
+	store Storer
+}
+
+func (suite *StoreSuite) TestCreate() {
+	a := assert.New(suite.T())
+	id := suite.store.Create(Puppy{Breed: "Wolf", Color: "Grey", Value: "450"})
+
+	a.Equal(id, 0)
+}
+
+func (suite *StoreSuite) TestCreateSecond() {
+	a := assert.New(suite.T())
+	id := suite.store.Create(Puppy{Breed: "Boxer", Color: "Brown", Value: "300"})
+
+	a.Equal(id, 1)
```

Comment:
I think what @anzdaddy was getting at here is that `TestCreate` should create 2 puppies, and check the second ones ID is incremented by 1 from the first, or at least different, think about what could be generating these id's if you don't know what the implementation is.
Otherwise, great tests.

Reply:
Done